introduction.md

```diff
@@ -0,0 +1,144 @@
+
+To by-pass the spend policy enforcement, an attacker would need to compromise all co-signing servers or to compromise all watchtowers. In the former case, an attacker could alter the Spend TX after the Unvault time-lock expires. In the latter case, a malicious Spend TX would not trigger the broadcast of a Cancel TX.
+
+### Denial-of-service Risk
+
+As a consequence of the N-1 resilience model, there are multiple attacks against the practical usage of the funds:
+
+- signing refusal by a stakeholder
+- unjustified Cancel TX push
+- unjustified Emergency TX push
+- taking down a co-signing server
+- taking down the coordinator
+
+In addition, if the network becomes highly congested, high feerates would trigger nodes to increase their minimum accepted feerate. Unvault TXs that are prepared with an initial 253 sat/kWU feerate could fail to be propagated sufficiently. Managers would lose access to funds delegated to them until stakeholders can re-sign Unvault TXs with a higher feerate. 
+
+### Key-tree Risk
+
+The usage of a BIP32 unhardened derivation introduce even more reliance of the signing devices security. Since the chaincodes are exchanged and stored on the wallets (running on untrusted online devices) of each participant, leaking a single child private key would be fatal to a participant's entire key-tree. Therefore the leak of a single child private key of all N stakeholders would be equivalent to the loss of all the funds. Using unhardened derivation implies that there is no partial loss of funds in case of a key leak by every stakeholder.
+
+### Transaction Fee Risk
+
+A fundamental issue arises with pre-signed transactions commiting to the feerate well in advance of the time of broadcast. The feerate is unpredictable and a transaction with a low feerate may not be included in a block in time. 
+
+The Unvault TXs have an additional output that allows dynamically allocating fees through child-pays-for-parent (CPFP). 
+
+The Cancel, Emergency and Unvault Emergency TXs aren't compatible with CPFP-carve out (FIXME: Why?), nor with ANYONECANPAY|ALL signatures (because this enables pinning attacks). The safest solution is to prepare multiple variants of the same transactions with a broad distribution of feerates, from typical to extreme and with many in-between. 
+
+
+### Underlying Assumptions
+
+#### Bitcoin
+
+- assume users can propagate revocation transactions to miners.
+- assume miners will mine the transaction paying the most feerate, regardless of any other (meta)data.
+- assume a significant hashrate to prevent reorgs of depth higher than the unvault time-lock length.
```

Comment:
Yes.

Reply:
Altho it's more complex than that. We don't want "natural reorgs" to be long, but they still statistically can happen. And then there is the game theory as in: are the miners going to mine the next block instead of rewriting the chain?
It's not only a simple hashrate discussion.